index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <a href="#" class="logo">Welcome</a>
+        <Nav id="menu">
+        <img src="./Images/icon-menu.png" alt="Menu Icon">
+        </Nav>
+    </header>
+<section class="bg">
+    <h1 class="hero up">Hey there. I’m <br>Usama</h1>
+    <h1 class="hero down">I'm a software<br>developer</h1>
```

Comment:
- [Optional] Kindly consider changing this subtitle to a `<h2>` element or any other heading element. This is because your web page should only have one `<h1>` element. This is best for semantics.